dropshot/tests/test_openapi_fuller.json

```diff
@@ -414,6 +510,25 @@
           "x"
         ]
       },
+      "Error": {
+        "description": "Error information from a response.",
+        "type": "object",
+        "properties": {
+          "error_code": {
+            "type": "string"
+          },
```

Comment:
Is this nullable? I believe there are cases in Omicron where the error code comes back `null`.

https://github.com/oxidecomputer/omicron/blob/53c6dc75dc49365983c032a73761320fbfe7e4f7/nexus/tests/integration_tests/unauthorized.rs#L716
https://github.com/oxidecomputer/omicron/blob/d08038c470370b08b5351d3ea9f35364e15461cb/nexus/tests/integration_tests/authn_http.rs#L79